mods/d2k/maps/atreides-02a/atreides02a-AI.lua

```diff
@@ -25,10 +25,13 @@ HarkonnenInfantryTypes = { "light_inf" }
 
 ActivateAI = function()
 	IdlingUnits[Harkonnen] = { }
+	Defending[Harkonnen] = { }
+	AttackDelay[Harkonnen] = 6000 * DifficultyModifier[Difficulty]
+	TimeBetweenAttacks[Harkonnen] = 5000 * DifficultyModifier[Difficulty]
```

Comment:
this will be a pain to maintain. Setting magic values for a script that doesn't even exist here. Rather we should call the functions here